pkg/aws/policies.go

```diff
@@ -848,18 +881,27 @@ func (c *awsClient) DeleteInlineRolePolicies(role string) error {
 	return nil
 }
 
-func (c *awsClient) deleteAccountRolePolicies(role *string) error {
-	err := c.detachAttachedRolePolicies(role)
-	if err != nil {
-		return err
-	}
-	err = c.DeleteInlineRolePolicies(aws.StringValue(role))
+func (c *awsClient) deleteAccountRoleInlinePolicies(role *string) error {
+	err := c.DeleteInlineRolePolicies(aws.StringValue(role))
```

Comment:
I think this function can be removed and just call `DeleteInlineRolePolicies` directly

Reply:
Ah yeah not sure why I made that an entire separate function in the first place

Reply:
the function was already there, you just reduced it since it had a lot of unnecessary stuff, but it ended up becoming just a no-op wrapper :stuck_out_tongue: